lib/counter_culture.rb

```diff
@@ -13,7 +13,13 @@ module ActiveRecord
 
     module ClassMethods
       # this holds all configuration data
-      attr_reader :after_commit_counter_cache
+      def after_commit_counter_cache
+        config = @after_commit_counter_cache
+        if !config && superclass.respond_to?(:after_commit_counter_cache)
+          config = superclass.after_commit_counter_cache
+        end
+        config
+      end
```

Comment:
Does this work when there is a counter cache both on the super-class and the sub-class? It looks to me like the sub-class config might clobber the super-class config.